utils/doclint/generateDotnetApi.js

```diff
@@ -320,19 +320,37 @@ function generateNameDefault(member, name, t, parent) {
   let enumName = generateEnumNameIfApplicable(member, name, t, parent);
   if (!enumName && member) {
     if (member.kind === 'method' || member.kind === 'property') {
-      // this should be easy to name... let's call it the same as the argument (eternal optimist)
-      let probableName = `${parent.name}${translateMemberName(``, name, null)}`;
-      let probableType = additionalTypes.get(probableName);
-      if (probableType) {
-        // compare it with what?
-        if (probableType.expression != t.expression) {
-          throw new Error(`Non-matching types with the same name. Panic.`);
+      let names = [
+        parent.alias || parent.name,
```

Comment:
`alias` is always initialized with `name` by default for members so you can omit `|| parent.name` here and below

Reply:
This might be a problem with something I did around the code, but there *are* instances where this was `undefined` for me, hence using the `||`...